### ISSUE TITLE
YouTube has a new channel format, translated by Invidious but not by Piped

### ISSUE THREAD

**Issue by Cade66** [libredirect/libredirect#121]
I was searching for ITV on DuckDuckGo and was surprised to find : `https://www.youtube.com/itv`
The url lands on `https://www.youtube.com/itv` : it is not modified by YouTube

Up to now we knew YouTube channels as being noted by `youtube.com/channel/`, `youtube.com/user/`, `youtube.com/c/`
It appears YouTube accepts -- at least for some channels -- the url : youtube.com/[CHANNEL]

Redirecting YouTube to Invidious with the new format **works**, Invidious translates it, i.e. `https://www.youtube.com/itv` to `https://yewtu.be/channel/UCoEqHpK5VNVEBRvwL0EW8Gw` : no problem

Redirecting YouTube to Piped with the new format **fails**, Piped does not translate and, i.e. `https://www.youtube.com/itv` lands on `https://piped.kavin.rocks/itv` and is **not** resolved.
**But** `https://piped.kavin.rocks/c/itv` is handled correctly (whilst **not** `https://piped.kavin.rocks/channel/itv`)

In other words, if **not** `channel/` **nor** `user/` **nor** `c/` **nor** `playlist`, the redirection **requires**, for **_Piped_**, the addition of `c/`

Brand new as far as i'm concerned, and bothering.

**Comment by ManeraKai** [comment]
I prefer to have the Piped team implement this rather than us. I can implement it but it won't make sens as every frontend should support this.

**Comment by Cade66** [comment]
I had that in mind right after having posted. Indeed, it's up to Piped to fix this.
I'll close this post.
